Xendit/Xendit.swift

```diff
@@ -279,7 +264,7 @@ import Foundation
     private static let PRODUCTION_XENDIT_BASE_URL = "https://api.xendit.co";
     
     private static let TOKEN_CREDENTIALS_PATH = "credit_card_tokenization_configuration";
-    private static let CREATE_CREDIT_CARD_PATH = "credit_card_tokens";
+    private static let CREATE_CREDIT_CARD_PATH = "v2/credit_card_tokens";
```

Comment:
Let's make sure that this versioning pattern is the approach we want to take across the Xendit API, for consistency